README.md

```diff
@@ -151,39 +149,9 @@ See [this README](/tools) if you want to use Docker.
 
 Once you have installed nGraph bridge, you can use TensorFlow to train a neural network or run inference using a trained model. 
 
-Use TensorFlow with nGraph to classify an image using a [frozen model]. 
-
-Download the Inception v3 trained model and labels file:
-
-    wget https://storage.googleapis.com/download.tensorflow.org/models/inception_v3_2016_08_28_frozen.pb.tar.gz
-
-Extract the frozen model and labels file from the tarball:
+Use `infer_image.py` in the [examples] directory to classify an image.
 
-    tar xvf inception_v3_2016_08_28_frozen.pb.tar.gz
-
-Download the image file: 
-
-    wget https://github.com/tensorflow/tensorflow/raw/master/tensorflow/examples/label_image/data/grace_hopper.jpg
-
-Download the TensorFlow script:
-
-   wget https://github.com/tensorflow/tensorflow/raw/master/tensorflow/examples/label_image/label_image.py
-
-Modify the downloaded TensorFlow script to run TensorFlow with nGraph optimizations:
-
-    import ngraph_bridge
-    ...
-    config = tf.compat.v1.ConfigProto()
-    config_ngraph_enabled = ngraph_bridge.update_config(config)
-    sess = tf.compat.v1.Session(config=config_ngraph_enabled)
-
-Run the classification:
-
-    python label_image.py --graph inception_v3_2016_08_28_frozen.pb \
-        --image grace_hopper.jpg --input_layer=input \
-        --output_layer=InceptionV3/Predictions/Reshape_1 \
-        --input_height=299 --input_width=299 \
-        --labels imagenet_slim_labels.txt 
+    python infer_image.py
 
```

Comment:
Maybe add a note here saying that this will download the Inception v3 model and a sample image before printing the results.

One thing that'd be nice to call out is that the only change required to the sample script is
```
import ngraph_bridge
```